app/helpers/Evaluation/IExercise.php

```diff
@@ -37,9 +37,9 @@ function getExerciseTests(): Collection;
 
   /**
    * Get configuration belonging to this exercise.
-   * @return ExerciseConfig
+   * @return ExerciseConfig|null
    */
-  function getExerciseConfig(): ExerciseConfig;
+  function getExerciseConfig(): ?ExerciseConfig;
```

Comment:
Well... ExerciseConfig should never be null, this hint is in my opinion pointless

Reply:
And yet it happened in production not too long ago. That's why defensive programming is a thing.

Reply:
No... this did not happen... EnvironmentConfig was null, that is utterly different thing...

Reply:
Hm, okay. Still, defensive programming :smile:

Reply:
Well... I am not gonna force you to change it, it is quite pointless... but defensive programming has to be taken seriously to some point... otherwise all we could do was checking this kind of errors...